security.md

```diff
@@ -4,35 +4,39 @@ title: Reporting a Security Vulnerability
 permalink: /security/
 ---
 
-# Reporting a Security Vulnerability
+# Security Policy
 
-The WSO2 security team welcomes contributions from our user community, developers, and security researchers to reinforce our product security. The security team will be more than happy to assist you in such efforts.
+Ballerina project maintainers take security issues very seriously and all the vulnerability reports are treated with the highest priority and confidentiality.
 
-We strongly encourage you to report security vulnerabilities to our private security mailing list: [security@ballerina.io](mailto:security@ballerina.io) - first, before disclosing them in any public forums.
+- [Reporting a vulnerability](#reporting-a-vulnerability)
+- [Handling a vulnerability](#handling-a-vulnerability)
 
-This is a private mailing list where only members of the WSO2 internal security team are subscribed to, and is treated as top priority.
+## Reporting a vulnerability
 
-If you wish to send secure messages to [security@ballerina.io](mailto:security@ballerina.io), you may use the following key:
+Ensure you are using the latest Ballerina version before you run an automated security scan or perform a penetration test against them.
 
-security@ballerina.io: 0168 DA26 2989 0DB9 4ACD  8367 E683 061E 2F85 C381 [pgp.mit.edu](https://pgp.surfnet.nl/pks/lookup?op=vindex&fingerprint=on&search=0xE683061E2F85C381)
+If you have any concerns regarding the security aspects of the source code or any other resource in this repo or have uncovered a security vulnerability, we strongly encourage you to report that to our private and highly confidential security mailing list: **security@ballerina.io** first using the below key without disclosing them in any forums, sites, or other groups - public or private. 
```

Comment:
Why are we removing the mailto: link? Isn't it a good practice to have them for email addresses?

Reply:
Added it.